CMakeLists.txt

```diff
@@ -7,82 +7,109 @@ function(cat IN_FILE OUT_FILE)
   file(APPEND ${OUT_FILE} "${CONTENTS}")
 endfunction()
 
-set(FILE_VERSION 5.39)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+MESSAGE("CMAKE_MODULE_PATH: " ${CMAKE_MODULE_PATH})
+find_package(pcre2 REQUIRED)
+
+option(BUILD_SHARED_LIBS "build as shared library" ON)
+
+# set(FILE_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/file/src")
+set(LIBMAGIC_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/file/src")
+set(LIBMAGIC_TEST_DIR "${CMAKE_CURRENT_SOURCE_DIR}/file/tests")
+set(WIN_HEADERS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/win-headers")
+# set(PCRE2_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/pcre2/src")
+set(PCRE2_LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/pcre2/install_dir/lib")
+
+set(FILE_VERSION 5.38)
```

Comment:
That would be a downgrade of file to `5.38`, is there a specific reason for this?